config/core/build-configs.yaml

```diff
@@ -251,6 +251,29 @@ fragments:
       - 'CONFIG_MMC_SDHCI=y'
       - 'CONFIG_MMC_SDHCI_PCI=y'
       - 'CONFIG_MMC_SDHCI_ACPI=y'
+      - 'CONFIG_CHROME_PLATFORMS=y'
+      - 'CONFIG_CHROMEOS_LAPTOP=y'
+      - 'CONFIG_CHROMEOS_TBMC=y'
+      - 'CONFIG_DM_CRYPT=y'
+      - 'CONFIG_DM_THIN_PROVISIONING=y'
+      - 'CONFIG_DM_INIT=y'
+      - 'CONFIG_DM_VERITY=y'
+      - 'CONFIG_CFS_BANDWIDTH=y'
+      - 'CONFIG_USER_NS=y'
+      - 'CONFIG_BLK_DEV_NVME=y'
```

Comment:
These 3 option seems to not be needed also for the Chromebooks that currently are attached to the lab (grunt, coral, octopus)

Reply:
Except for `CONFIG_BLK_DEV_NVME` they are needed to boot Eve (Google Pixelbook), but since we do not appear to have that in the pipeline I guess we can drop these and I'll build my test kernels with local changes for Eve.

I do not know which Chromebooks come with NVME drives and we don't appear to have any in the pipeline so we can drop  `CONFIG_BLK_DEV_NVME` as well.

@sonnydeez Can you tell us for which specific boards did you add this config initially?

Reply:
Since we do not have any boards with NVME I'm marking this as resolved. When we will add a device with NVME we can also add this config according to the principles discussed in this review. Thanks everyone! :)